feat: 로또 당첨 결과 대조

### 2. 로또 당첨 번호
- [x] 중복되지 않는 숫자(당첨 번호) 6개와 + 보너스 번호 1개가 입력된다.

### 3. 로또 당첨 결과
- [x] 당첨은 1등부터 5등까지 있다.
- [x] 당첨 기준과 금액은 아래와 같다.
> - 1등 / 6개 번호 일치 / 2,000,000,000원 <br>
> - 2등 / 5개 번호 + 보너스 번호 일치 / 30,000,000원 <br>
> - 3등 / 5개 번호 일치 / 1,500,000원 <br>
> - 4등 / 4개 번호 일치 / 50,000원 <br>
> - 5등 / 3개 번호 일치 / 5,000원 <br>

### 5. 로또 머신
- [x] 구매한 로또 번호와 당첨 번호를 비교한다.

### 6. 입출력
#### 입력
- [x] 당첨 번호와 보너스 번호를 입력 받는다.
    - [x] 당첨 번호는 쉼표(,)를 기준으로 구분한다.
    - [x] 당첨 번호를 입력받은 후에 보너스 번호를 입력 받는다.
```
당첨 번호를 입력해 주세요.
1,2,3,4,5,6

보너스 번호를 입력해 주세요.
7
```

#### 출력
- [x] 당첨 내역을 아래와 같이 출력한다.
```
당첨 통계
---
3개 일치 (5,000원) - 1개
4개 일치 (50,000원) - 0개
5개 일치 (1,500,000원) - 0개
5개 일치, 보너스 볼 일치 (30,000,000원) - 0개
6개 일치 (2,000,000,000원) - 0개
```
- [x] 수익률은 소수점 둘째 자리에서 반올림한다. (ex. 100.0%, 51.5%, 1,000,000.0%)
- [ ] 수익률 값은 천 단위마다 `,`로 구분한다. (ex. 1,000,000.0%)
```
총 수익률은 62.5%입니다.
```

### 예외처리
- 사용자가 잘못된 값을 입력할 경우 `IllegalArgumentException`를 발생시키고, "[ERROR]"로 시작하는 에러 메시지를 출력한다.
```
[ERROR] 로또 번호는 1부터 45 사이의 숫자여야 합니다.
```
- `Exception`이 아닌 `IllegalArgumentException`, `IllegalStateException` 등과 같은 명확한 유형을 처리한다.

#### 예외처리 유형
- [x] 1~45 범위 바깥의 정수를 입력한 경우
- [x] 입력한 당첨 번호 6개 + 보너스 번호 1개가 서로 중복되는 경우

--- a/src/main/java/lotto/view/OutputView.java
+++ b/src/main/java/lotto/view/OutputView.java
@@ -1,5 +1,6 @@
 package lotto.view;
 
+import java.text.DecimalFormat;
 import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
@@ -18,8 +19,14 @@ public class OutputView {
         System.out.printf(Message.OUTPUT_COMPLETE_BUY_LOTTO.message, lotties.size());
         for (Lotto lotto : lotties) {
             List<Integer> numbers = lotto.getNumbers();
-            System.out.printf(Message.OUTPUT_LOTTO_NUMBERS.message, formatter(numbers));
+            System.out.printf(Message.OUTPUT_LOTTO_NUMBERS.message, joinFormatter(numbers));
         }
+    }
+
+    private String joinFormatter(List<Integer> numbers) {
+        return numbers.stream()
+                .map(String::valueOf)
+                .collect(Collectors.joining(DIGIT_COMMA_WITH_BLANK));
     }
 
     public void outputLottoResults(List<LottoRank> rankResults) {
@@ -39,13 +46,13 @@ public class OutputView {
     }
 
     public void outputProfitRate(double profitRate) {
-        System.out.printf(Message.OUTPUT_PROFIT_RATE_BY_PERCENT.message, profitRate);
+        String formattedRate = thousandFormatter(profitRate);
+        System.out.printf(Message.OUTPUT_PROFIT_RATE_BY_PERCENT.message, formattedRate);
     }
 
-    private String formatter(List<Integer> numbers) {
-        return numbers.stream()
-                .map(String::valueOf)
-                .collect(Collectors.joining(DIGIT_COMMA_WITH_BLANK));
+    private String thousandFormatter(double profitRate) {
+        DecimalFormat decimalFormat = new DecimalFormat("#,###");
+        return decimalFormat.format(profitRate);
     }
 
     private enum Message {

